app/src/androidTest/java/testapp/NavigationViewTest.java

```diff
@@ -3,54 +3,100 @@
 import com.mapbox.api.directions.v5.models.DirectionsRoute;
 import com.mapbox.services.android.navigation.testapp.test.TestNavigationActivity;
 import com.mapbox.services.android.navigation.ui.v5.NavigationViewOptions;
+import com.mapbox.services.android.navigation.ui.v5.listeners.SpeechAnnouncementListener;
 import com.mapbox.services.android.navigation.ui.v5.map.NavigationMapboxMap;
+import com.mapbox.services.android.navigation.ui.v5.voice.SpeechAnnouncement;
 import com.mapbox.services.android.navigation.v5.navigation.MapboxNavigation;
-
 import org.junit.Test;
-
 import testapp.activity.BaseNavigationActivityTest;
 
-import static junit.framework.Assert.assertNotNull;
+import static junit.framework.Assert.*;
 import static testapp.action.NavigationViewAction.invoke;
 
-public class NavigationViewTest extends BaseNavigationActivityTest {
+public class NavigationViewTest extends BaseNavigationActivityTest implements SpeechAnnouncementListener {
+
+    @Override
```

Comment:
I think your codestyles are mismatched with the project hense all the line changes here, you might want to fix that and resubmit